app/views/new_filters/_salary_filter.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="govuk-form-group app-filter__group" data-qa="filters__funding">
+  <legend class="govuk-fieldset__legend govuk-fieldset__legend--s">
+    <h3 class="govuk-fieldset__heading">Salary</h3>
+  </legend>
+
+  <div class="govuk-checkboxes govuk-checkboxes--small">
+    <div class="govuk-checkboxes__item" data-qa="subject">
+      <%= form.check_box(
```

Comment:
can we just use govuk_check_box here?

Reply:
https://github.com/DFE-Digital/find-teacher-training/pull/650#discussion_r575252853